mobile/src/main/java/mycroft/ai/activities/SettingsActivity.java

```diff
@@ -217,6 +235,46 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 startActivity(new Intent(getActivity(), SettingsActivity.class));
                 return true;
             }
+
+            return super.onOptionsItemSelected(item);
+        }
+    }
+
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    public static class AboutPreferenceFragment extends PreferenceFragment {
+        @Override
+        public void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            addPreferencesFromResource(R.xml.pref_about);
+            setHasOptionsMenu(true);
+
+            bindPreferenceSummaryToValue(findPreference("versionName"));
```

Comment:
I think these strings should be constants, especially since they need to be identical to those in MainActivity.